From f82c25a9637fabcbb4c85cb3e0145a99b027d722 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Tue, 23 Aug 2016 18:44:08 +0300
Subject: [PATCH 0446/1345] doc: apn806: Add eMMC clock to the documentation

commit  3d9bf3517a88e416e257754f7ad932f0863400d3 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Add fixed eMMC clock to the list of AP system controller
clocks list.
Update clock names.

Change-Id: Idada8b086e492331bade781d402bfa23ca14ed4b
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32146
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm/marvell/ap806-system-controller.txt        |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/arm/marvell/ap806-system-controller.txt b/Documentation/devicetree/bindings/arm/marvell/ap806-system-controller.txt
index a29a07e..e951241 100644
--- a/Documentation/devicetree/bindings/arm/marvell/ap806-system-controller.txt
+++ b/Documentation/devicetree/bindings/arm/marvell/ap806-system-controller.txt
@@ -14,6 +14,7 @@ a number of clocks:
  - 1: clock of CPU cluster 1
  - 2: fixed PLL at 1200 Mhz
  - 3: MSS clock, derived from the fixed PLL
+ - 4: eMMC clock, derived from the fixed PLL
  - 5: AP HCLK, derived from sample-at-reset configuration.
 
 Required properties:
@@ -23,7 +24,7 @@ Required properties:
  - reg: register area of the AP806 system controller
  - #clock-cells: must be set to 1
  - clock-output-names: must be defined to:
-    "ap-cpu-cluster-0", "ap-cpu-cluster-1", "ap-fixed", "ap-mss", "ap-hclk"
+   "ap-cpu-cluster-0", "ap-cpu-cluster-1", "ap-fixed", "ap-mss", "ap-emmc", "ap-hclk"
 
 Example:
 
@@ -31,6 +32,6 @@ Example:
 		compatible = "marvell,ap806-system-controller", "syscon";
 		#clock-cells = <1>;
 		clock-output-names = "ap-cpu-cluster-0", "ap-cpu-cluster-1",
-				     "ap-fixed", "ap-mss", "ap-hclk";
+				     "ap-fixed", "ap-mss", "ap-emmc", "ap-hclk";
 		reg = <0x6f4000 0x1000>;
 	};
-- 
1.7.9.5

